django/db/models/functions/datetime.py

```diff
@@ -87,6 +87,13 @@ class ExtractDay(Extract):
     lookup_name = 'day'
 
 
+class ExtractWeek(Extract):
+    """
+    Returns 1-53 based on ISO-8601, i.e., Monday is the first of the week.
```

Comment:
I'm indifferent to this, but it seems like a reasonable thing to mention. Most likely, MySQL's documentation influenced me, as 1-53 is in there.